scripts/remove

```diff
@@ -33,7 +33,7 @@ ynh_system_user_delete $app
 sudo systemctl reload nginx
 
 # Remove app dependencies
-ynh_remove_app_dependencies "$PKG_DEPENDENCIES"
+ynh_remove_app_dependencies $app
```

Comment:
I'm sorry, I promise I don't want to be such a pain, but you don't need ANY parameter: $app is a global variable that's already been set :innocent: